packages/shared/src/components/modals/common/ModalTabs.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import React, { ReactElement, useContext } from 'react';
+import { ModalPropsContext } from './types';
+
+export function ModalTabs(): ReactElement {
+  const { activeTab, onTabChange, tabs } = useContext(ModalPropsContext);
+  return (
+    <ul className="flex flex-row gap-4">
+      {tabs.map((tab) => (
+        <button
+          key={tab}
+          className={classNames(
+            'relative py-1.5 px-3 text-center typo-callout rounded-6',
+            tab === activeTab
+              ? 'bg-theme-bg-pepper font-bold'
+              : 'text-theme-label-tertiary',
+          )}
+          onClick={() => onTabChange(tab)}
+          type="button"
+          role="menuitem"
+        >
+          {tab}
+          {tab === activeTab && (
+            <div
+              className="absolute mx-auto w-4 h-px bg-theme-label-primary"
+              style={{ bottom: '-0.75rem', left: 'calc(50% - 0.5rem)' }}
+            />
```

Comment:
Is this not just the active state of a button?
Pretty sure it should be a class on the button above rather then a extra element.

Reply:
The active state has a background and a border below it. Thats why this is a separate element
<img width="421" alt="image" src="https://user-images.githubusercontent.com/1488164/201073090-ef97b704-fd99-4e93-b734-b3b2e40452d5.png">


The design of the tabs has been changed slightly
https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/a/agogLmZ

Reply:
Oh right!
Sorry missed that thing indeed 🙏